corehq/apps/accounting/invoicing.py

```diff
@@ -491,6 +491,17 @@ def unit_cost(self):
 class UserLineItemFactory(FeatureLineItemFactory):
 
     @property
+    def unit_cost(self):
+        non_prorated_unit_cost = super(UserLineItemFactory, self).unit_cost
+        if self.is_prorated:
+            return Decimal(
+                "%.2f" % round(
+                    non_prorated_unit_cost * self.num_prorated_days / self._days_in_billing_period, 2
```

Comment:
Are you sure this isn't ever doing integer division?  I think as long as `non_prorated_unit_cost` is a Decimal it should behave as expected.